File: msgraph-mail/typescript/src/users/followupFlagStatus.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class FollowupFlagStatus implements Parsable<FollowupFlagStatus> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: FollowupFlagStatus, node: ParseNode) => void> {
        return new Map<string, (item: FollowupFlagStatus, node: ParseNode) => void>([
        ]);
    };
}

File: msgraph-mail/dotnet/Users/FollowupFlagStatus.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users {
    public class FollowupFlagStatus : IParsable<FollowupFlagStatus> {
        public IDictionary<string, Action<FollowupFlagStatus, IParseNode>> DeserializeFields => new Dictionary<string, Action<FollowupFlagStatus, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}
